src/AutoRest.CSharp/Mgmt/AutoRest/MgmtConfiguration.cs

```diff
@@ -421,6 +427,9 @@ internal static MgmtConfiguration LoadConfiguration(JsonElement root)
             var prependRPPrefixList = prependRPPrefix.ValueKind == JsonValueKind.Array
                 ? prependRPPrefix.EnumerateArray().Select(t => t.ToString()).ToArray()
                 : Array.Empty<string>();
+            var disableResourceReturnList = disableResourceReturn.ValueKind == JsonValueKind.Array
+                ? prependRPPrefix.EnumerateArray().Select(t => t.ToString()).ToArray()
```

Comment:
This is incorrect

Reply:
My bad, I just cope-paste and forgot to change it ==